cardinal/transaction.go

```diff
@@ -44,6 +44,10 @@ func NewTransactionTypeWithEVMSupport[Msg, Result any](name string) *Transaction
 	}
 }
 
+func (t *TransactionType[Msg, Result]) AddToQueue(world *World, data Msg, sigs ...*sign.SignedPayload) {
+	ecs.AddToQueue[Msg, Result](world.implWorld, t.impl, data, sigs...)
```

Comment:
Can't you just make this t.impl.AddToQueue and leave out the change to cardinal/ecs/world.go?

Reply:
Yeah, great idea, done!